extensions/ql-vscode/package.json

```
@@ -840,8 +840,8 @@
         "title": "CodeQL: Choose Database from Folder"
       },
       {
-        "command": "codeQL.chooseMultipleDatabaseFolder",
-        "title": "CodeQL: Choose Folder contains all Database Folders to import"
+        "command": "codeQL.chooseDatabaseFoldersParent",
+        "title": "CodeQL: Choose Folder to import all databases contained in it"
```

I think this is better:

```suggestion
        "title": "CodeQL: Choose Parent Folder and import all databases directly contained in it"
```